app/dao/fact_billing_dao.py

```diff
@@ -74,7 +74,7 @@ def fetch_usage_for_all_services_sms(start_date, end_date):
         Service.name
     )
 
-    return query.all()
```

Comment:
Optional: Also for comparison `fetch_usage_for_all_services_letter` still calls `.all` so there is a bit of inconsistency between the `fetch_usage_for_all_services_x` functions and what they return. I can tolerate this though or maybe a slight change to the function name to say it returns a query?

Reply:
yeah, to confirm, calling `iter(query)` executes it in the same way as `.all`. It doesn't functionally make any difference, it may in fact have very marginal performance improvements or RAM usage improvements by not calling `.all` since it might copy data about a little bit less, but i wouldn't expect that to be noticeable.

i'll take a look at the usage of `_letter`. happy to keep them all consistent.